### PR TITLE
pythonPackages.asyncssh: init at 1.13.0

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
I needed this

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/asyncssh/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonOlder, isPy34, isPy35, isPy36
+, cryptography
+, bcrypt, libsodium, libnacl, nettle, pyopenssl }:
+
+buildPythonPackage rec {
+  pname = "asyncssh";
+  version = "1.12.2";
+  disabled = pythonOlder "3.4";
+
+  doCheck = true;
```
Comment (FRidh) [inline pkgs/development/python-modules/asyncssh/default.nix]: this is the default, please drop

**Comment by FRidh** [comment]
@GrahamcOfBorg build python2.pkgs.asyncssh python3.pkgs.asyncssh

**Comment by worldofpeace** [comment]
@FRidh Can I just kill it :)

File: pkgs/development/python-modules/asyncssh/default.nix (hunk 1 above)
Comment (worldofpeace) [reply pkgs/development/python-modules/asyncssh/default.nix]: Thoroughly dropped

**Comment by worldofpeace** [comment]
@FRidh This works now :+1:

**Comment by worldofpeace** [comment]
@FRidh Can you check if this builds? I've made all requested changes and then some.

File: pkgs/development/python-modules/asyncssh/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    bcrypt
+    cryptography
+    gssapi
+    libnacl
+    libsodium
+    nettle
+    pyopenssl
+  ];
+
+  patches = [ ./ASYNCSSH-1.12.2_sspi-stub.patch ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/asyncssh/default.nix]: I'd prefer deleting the file using `rm` in `postPatch`. And please add a comment why it is deleted.

File: pkgs/development/python-modules/asyncssh/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonOlder, isPy34, isPy35, isPy36
```
Comment (dotlambda) [inline pkgs/development/python-modules/asyncssh/default.nix]: isPy34, isPy35, isPy36 are not used

Comment (worldofpeace) [reply pkgs/development/python-modules/asyncssh/default.nix]: Removed

File: pkgs/development/python-modules/asyncssh/default.nix (hunk 2 above)
Comment (worldofpeace) [reply pkgs/development/python-modules/asyncssh/default.nix]: Done

**Comment by worldofpeace** [comment]
Just saw that this package is now old.

**Comment by dotlambda** [comment]
@worldofpeace In that case, update this PR to use the most recent version.

@GrahamcOfBorg build python3.pkgs.asyncssh

**Comment by dotlambda** [comment]
In order to fix

> OK (skipped=14)
/bin/sh: openssl: not found
/bin/sh: ssh: not found

we'd probably have to add `openssl` and `openssh` to `checkInputs`. But we don't need to because the tests seem to be skipped when the executables are not present.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python3.pkgs.asyncssh

**Comment by dotlambda** [comment]
Thanks!

**Comment by worldofpeace** [comment]
Yay! Thank you for the review.

**Pull request merged**